android-ktx/build.gradle

```diff
@@ -9,6 +9,7 @@ plugins {
 }
 
 repositories {
+    mavenLocal()
```

Comment:
Leftover ? We try not to let this to prevent eventual errors.

Reply:
Since we publish only snapshots in maven local why would that be a problem?